.goreleaser.yaml

```diff
@@ -0,0 +1,57 @@
+version: 2
+
+release:
+  name_template: Test Splitter v{{.Version}}
+  draft: true
```

Comment:
We may consider publishing the GitHub release instead of creating a draft.
Thought? @wooly

Reply:
What's the difference between draft and when github marks it as pre-release, is there one?

Reply:
Draft release is not accessible by the public, and git `tag` will not be created for that release.

Reply:
I think my question is if we want to publish draft manually, or automate that. We've been automatically publish the draft, so maybe we should keep that behavior?

Reply:
Ah, I reckon we don't need to produce a draft release? We just unblock a new patch or patch rc when we want to produce a pre-release version